swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md

```diff
@@ -199,7 +199,7 @@ Now, you tested a simple Ballerina HTTP server application for GraalVM compatibi
 
 ## Create GraalVM-compatible library packages
 
-Packages that solely use Ballerina standard libraries and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
+Packages that solely use Ballerina library packages and connectors or those that use platform Java libraries provided by the distribution are inferred to be compatible with GraalVM.
```

Comment:
Connectors (i.e. ballerinax packages) are now part of Ballerina library.

Reply:
Changed to `Packages that solely use Ballerina library packages`.